script.js

```diff
@@ -94,10 +94,11 @@ var Typer={
       console.html(text.replace(rtn,"<br/>").replace(rtt,"&nbsp;&nbsp;&nbsp;&nbsp;").replace(rts,"&nbsp;"));// replace newline chars with br, tabs with 4 space and blanks with an html blank
 			window.scrollBy(0,50); // scroll to make sure bottom is always visible
 		}
-		if ( key.preventDefault && key.key !== 'F11' ) { // prevent F11(fullscreen) from being blocked
+    var isNotFnKey = !isFunctionKey(key.key);
```

Comment:
I appreciate the idea but this function doesn't seem right.

```
isFunctionKey("F1") === false
isFunctionKey("F10") === false
isFunctionKey("A") === false
```

It can probably be done with a simple regex like 
```
var isNotFnKey = !key.key.match(/^F\d[0-2]?$/m);
```